spec/game_logic_spec.rb

```diff
@@ -0,0 +1,61 @@
+require './lib/game_logic'
```

Comment:
You can use `requre_relative` instead require  for files in your directory

Reply:
Thank you for this!!

It has been handled.